src/components/UniversalNavbarExpanded/DropdownNav/DropdownCta.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+// Reused assets from UniversalNavbar
+import { DropdownLinkIcon } from '../../UniversalNavbar/assets/icons.js'
+
+// EDS core components
+import { Spacer, Body, COLORS, TitleSmall } from '../../index'
+
+// Helpers
+import { removeLastWord, getLastWord } from '../../../helpers/splitLastWord'
+
+// Styles
+import styles from './DropdownCta.module.scss'
+
+/**
+ * CTA copy for use in DropdownNav
+ *
+ *
+ * @param {string} title - A short title
+ * @param {string} subcopy - A sentence of supporting copy
+ *
+ * @return {JSX.Element}
+ */
+const DropdownCta = ({ title, subcopy }) => (
+  <>
+    <TitleSmall.Serif.Book500>
+      <span>{removeLastWord(title)}</span>
```

Comment:
Can we add a comment or create a shared component which references the function (maintaining one line text on resize)

Reply:
b4dd2bf

Reply:
7c6efbf